docs/index.rst

```diff
@@ -44,9 +44,10 @@ More information and guidances can be found in the:
   how-to-guides/priority
   how-to-guides/pci-ssn-pii-detection
   how-to-guides/pci-ssn-pii-redaction
+  how-to-guides/client-supplied-encryption.md
```

Comment:
@Daniel085  This might work, but let's be consistent, we're referencing everything else without .md